liwords-ui/src/gameroom/table.tsx

```diff
@@ -138,7 +138,7 @@ export const Table = React.memo((props: Props) => {
 
   useEffect(() => {
     if (gameContext.playState === PlayState.GAME_OVER || isObserver) {
-      return () => {};
+      return;
```

Comment:
my linter complained about not returning a function (consistent returns). Do we have the same linter settings? They should be checked into this repo.